admin/partials/admin-monk-select-render.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * Provide the view for the monk_select_render function
+ *
+ * @link       https://github.com/brenoalvs/monk
+ * @since      1.0.0
+ *
+ * @package    Monk
+ * @subpackage Monk/Admin/Partials
+ */
+
+	$options = get_option( 'monk_language' );
+	?>
+	<select name="monk_language[language_select]">
+		<option value="portuguese"<?php selected( $options['language_select'], "portuguese" ); ?>>Portuguese</option>
```

Comment:
In PHP functions use single quotes.
Remember in HTML = double quotes, PHP = single quotes